src/test/java/peoplesoft/model/money/PaymentTest.java

```diff
@@ -0,0 +1,220 @@
+package peoplesoft.model.money;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static peoplesoft.testutil.Assert.assertThrows;
+import static peoplesoft.testutil.TypicalPersons.ALICE;
+import static peoplesoft.testutil.TypicalPersons.BENSON;
+import static peoplesoft.testutil.TypicalPersons.CARL;
+import static peoplesoft.testutil.TypicalPersons.getTypicalAddressBook;
+
+import java.time.Duration;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.junit.jupiter.api.Test;
+
+import peoplesoft.model.AddressBook;
+import peoplesoft.model.Model;
+import peoplesoft.model.ModelManager;
+import peoplesoft.model.UserPrefs;
+import peoplesoft.model.employment.Employment;
+import peoplesoft.model.job.Job;
+import peoplesoft.model.money.exceptions.PaymentRequiresPersonException;
+import peoplesoft.model.person.Person;
+import peoplesoft.model.util.ID;
+import peoplesoft.testutil.PersonBuilder;
+
+
+public class PaymentTest {
+
+    private static final Job EATING = new Job(new ID(1043), "Eating", Duration.ofDays(1));
+    private static final Job RUNNING = new Job(new ID(3175), "Running", Duration.ofHours(8));
+
+    @Test
+    public void createPendingPayments_throwsPaymentRequiresPersonException() {
```

Comment:
i think the test name might be a bit ambiguous (and might imply that createPendingPayments should always throw that exception), maybe sth like this instead?

```suggestion
    public void createPendingPayments_noPersonAssigned_throwsPaymentRequiresPersonException() {
```